gitlab-pipeline/stage/test.yml

```diff
@@ -623,20 +614,41 @@ test_full_integration_enterprise:
 
     # Output storage io stats
     - df -h . | tail -1 | awk '{system("hdparm -tT "$1);}'
+
     # Get and install the integration test requirements
     - apk add $(cat ${WORKSPACE}/integration/tests/requirements/apk-requirements.txt)
-    - pip install  -r ${WORKSPACE}/integration/tests/requirements/python-requirements.txt
-    - pip3 install -r ${WORKSPACE}/integration/tests/requirements/python-requirements.txt
+    - pip install -r ${WORKSPACE}/integration/tests/requirements/python-requirements.txt
```

Comment:
Very strange... are you missing a rebase?

Reply:
Probs, I'll fix it